maquetado/setup/index.html

```diff
@@ -9,15 +9,100 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Front-End Tools</h1>
-  <div class="tech">
-    <img class="tech-logo" src="assets/react_icon.svg" alt="ReactJS" />
-    <h2 class="tech-title">ReactJS</h2>
-    <h4 class="tech-date">Release:2013</h4>
-    <p class="tech-text">
-      React is a declarative, component-based
-       JavaScript library for building user interfaces.
-     </p>
+  <div class="title-techs">
+    <h1 class="title">Front-End Tools</h1>
```

Comment:
Remember, there is only is only one `h1` per section, as you are not using sections right now, this should be an `h2`